dotnet/README.MD

```diff
@@ -0,0 +1,179 @@
+ [free-trial]: https://cloud.google.com/free-trial
+ 
+ 
+ ## Before you begin
+ 
+ This sample assumes you have [.NET Core 2.0][.NET Core] 2.0 installed.
+ 
+ [.NET Core]:https://docs.microsoft.com/en-us/dotnet/core/
+ 
+ To install .NET Core please follow [this][this] tutorial.
+ 
+ [this]:https://www.microsoft.com/net/learn/get-started/windows 
+ 
+ ### Creating a Project in the Google Cloud Platform Console
+ 
+ If you haven't already created a project, create one now. Projects enable you to
+ manage all Google Cloud Platform resources for your app, including
+ deployment,
+ access control, billing, and services.
+ 
+ 1. Open the [Cloud Platform Console][cloud-console].
+ 1. In the drop-down menu at the top, select **Create a project**.
+ 1. Give your project a name.
+ 1. Make a note of the project ID, which might be different from the project
+    name. The project ID is used in commands and in configurations.
+ 
+ [cloud-console]: https://console.cloud.google.com/
+ 
+ ### Enabling billing for your project.
+ 
+ If you haven't already enabled billing for your project, [enable
+ billing][enable-billing] now.  Enabling billing allows is required to use
+ Cloud Bigtable and to create VM instances.
+ 
+ [enable-billing]: https://console.cloud.google.com/project/_/settings
+ 
+ ### Enable the Cloud Bigtable APIs.
+ 
+ Make sure to [enable the Bigtable APIs][enable-bigtable-api].
+ 
+ [enable-bigtable-api]: https://console.cloud.google.com/apis/library?q=bigtable
+ 
+ ### Install the Google Cloud SDK.
+ 
+ If you haven't already installed the Google Cloud SDK, [install the Google
+ Cloud SDK][cloud-sdk] now. The SDK contains tools and libraries that enable you
+ to create and manage resources on Google Cloud Platform.
+ 
+ [cloud-sdk]: https://cloud.google.com/sdk/
+ 
+ ### Setting Google Application Default Credentials
+ 
+ Set your [Google Application Default
+ Credentials][application-default-credentials] by [initializing the Google Cloud
+ SDK][cloud-sdk-init] with the command:
+ 
+     gcloud init
+ 
+ Generate a credentials file by running the [application-default login](https://cloud.google.com/sdk/gcloud/reference/auth/application-default/login) command:
+ 
+     gcloud auth application-default login
+ 
+ [cloud-sdk-init]: https://cloud.google.com/sdk/docs/initializing
+ [application-default-credentials]: https://developers.google.com/identity/protocols/application-default-credentials
+ 
+ 
+ ## Provisioning an instance
+ 
+ Follow the instructions in the [user
+ documentation](https://cloud.google.com/bigtable/docs/creating-instance) to
+ create a Google Cloud Platform project and Cloud Bigtable instance if necessary.
+ You'll need to reference your project id and instance id to run the
+ application.
+ 
+ 
+ ## Running the application
+ 
+ In appsettings.json replace `[PROJECT_ID]` with google cloud project name and `[BIGTABLEINSTANCE]` with what you called it when provisioning the instance.
+ 
+    "projectId": `[PROJECT_ID]`,
```

Comment:
Since this needs to be a valid JSON sample, maybe it should be more like:

```json
"projectId": "<GCP-project-id>",
"instanceId": "<Bigtable-instance-id>",
```

Because "`" is not a valid quoting character in JSON. Thoughts?